.vitepress/theme/styles/zh-cn.css

```diff
@@ -7,9 +7,12 @@
 }
 
 body {
-  --vt-font-family-base: Quotes, Inter, -apple-system, BlinkMacSystemFont,
+  --vt-font-family-base: Quotes, 'Inter var experimental', 'Inter var', -apple-system, BlinkMacSystemFont,
```

Comment:
Looks like these 2 fonts are not defined in vuejs/theme (used as doc theme) but vitepress/theme-default

Reply:
It works (win10) btw

Reply:
https://github.com/vuejs/vitepress/pull/777   

Is it related to this?

Reply:
> [vuejs/vitepress#777](https://github.com/vuejs/vitepress/pull/777)
> 
> Is it related to this?

Looks like it is. But now can we remove the `Inter` font instead of using `'Inter var experimental', 'Inter var'`? Since it's not defined in vuejs/theme, I think there's no difference. And it's also better to add some comments on this.

We can rise PR to vuejs/theme later for these changes, and sync back here after they works.

Reply:
I did this at the beginning, but consider whether deleting Inter fonts will have other effects.